src/res/translation/translation_zh_CN.ts

```diff
@@ -0,0 +1,3072 @@
+    </message>
+    <message>
+        <location filename="../../aboutdlgbase.ui" line="72"/>
+        <source>Copyright (C) 2005-2021 The Jamulus Development Team</source>
+        <translation>版权所有 (C) 2005-2021 Jamulus 开发团队</translation>
+    </message>
+    <message>
+        <location filename="../../aboutdlgbase.ui" line="106"/>
+        <source>A&amp;bout</source>
+        <translation>关于(&amp;A)</translation>
+    </message>
+    <message>
+        <location filename="../../aboutdlgbase.ui" line="120"/>
+        <source>&amp;Libraries</source>
+        <translation>依赖库(&amp;L)</translation>
+    </message>
+    <message>
+        <location filename="../../aboutdlgbase.ui" line="134"/>
+        <source>&amp;Contributors</source>
+        <translation>贡献者(&amp;C)</translation>
+    </message>
+    <message>
+        <location filename="../../aboutdlgbase.ui" line="148"/>
+        <source>&amp;Translation</source>
+        <translation>翻译者(&amp;T)</translation>
+    </message>
+    <message>
+        <location filename="../../aboutdlgbase.ui" line="198"/>
+        <source>&amp;OK</source>
+        <translation>确认(&amp;O)</translation>
+    </message>
+</context>
+<context>
+    <name>CAnalyzerConsole</name>
+    <message>
+        <location filename="../../analyzerconsole.cpp" line="47"/>
+        <source>Analyzer Console</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../analyzerconsole.cpp" line="65"/>
+        <source>Error Rate of Each Buffer Size</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
+    <name>CAudioMixerBoard</name>
+    <message>
+        <location filename="../../audiomixerboard.cpp" line="849"/>
+        <source>Personal Mix at the Server</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../audiomixerboard.cpp" line="850"/>
+        <source>When connected to a server, the controls here allow you to set your local mix without affecting what others hear from you. The title shows the server name and, when known, whether it is actively recording.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../audiomixerboard.cpp" line="923"/>
+        <source>Server</source>
+        <translation>服务器</translation>
+    </message>
+    <message>
+        <location filename="../../audiomixerboard.cpp" line="932"/>
+        <source>T R Y I N G   T O   C O N N E C T</source>
+        <translation>尝 试 连 接 中</translation>
+    </message>
+    <message>
+        <location filename="../../audiomixerboard.cpp" line="1102"/>
+        <source>RECORDING ACTIVE</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../../audiomixerboard.cpp" line="1112"/>
+        <source>Personal Mix at: </source>
```

Comment:
This one should be translated before a merge since it's prominent on the mixing board.